src/pint/observatory/clock_file.py

```diff
@@ -864,7 +872,11 @@ def evaluate(self, t, limits="warn"):
         corrections : astropy.units.Quantity
             The corrections in units of microseconds.
         """
-        if np.any(t > self.clock_file.time[-1]):
+        with warnings.catch_warnings():
+            # FIXME: could these warnings be useful?
+            warnings.filterwarnings("ignore", r".*dubuious year", erfa.ErfaWarning)
```

Comment:
typo!

Reply:
Since the warning suppression wasn't working because of the typo, and the test suite passed anyway, clearly it wasn't needed here. (I feel better not suppressing the warning in library code unless I can clearly justify the need to.)